docs/user/hbase/install.rst

```diff
@@ -23,50 +25,110 @@ and untar it somewhere convenient:
+A few suggested configurations are below:
 
 .. tabs::
 
+    .. group-tab:: Amazon EMR
+
+        When using EMR to install HBase or Hadoop there are AWS specific jars that need to be used (e.g. EMR FS).
+        It is recommended to use EMR to install Hadoop and/or HBase or order to properly configure and install these
+        dependencies (especially when using HBase on S3).
+
+        If you used EMR to install Hadoop and HBase, you can view their classpaths using the ``hadoop classpath`` and
+        ``hbase classpath`` commands to build an appropriate classpath to include jars and configuration files for
+        GeoMesa HBase:
+
+        .. code-block:: bash
+
+            export GEOMESA_HADOOP_CLASSPATH=$(hadoop classpath)
+            export GEOMESA_HBASE_CLASSPATH=$(hbase classpath)
+            export GEOMESA_HBASE_HOME=/opt/geomesa
+            export PATH="${PATH}:${GEOMESA_HBASE_HOME}/bin"
+
     .. group-tab:: Standard
 
         Configure GeoMesa to use pre-installed HBase and Hadoop distributions:
 
         .. code-block:: bash
 
-            $ export HADOOP_HOME=/path/to/hadoop
-            $ export HBASE_HOME=/path/to/hbase
-            $ export GEOMESA_HOME=/opt/geomesa
-            $ export PATH=$PATH:$GEOMESA_HOME/bin
+            export HADOOP_HOME=/path/to/hadoop
+            export HBASE_HOME=/path/to/hbase
+            export GEOMESA_HBASE_HOME=/opt/geomesa
+            export PATH="${PATH}:${GEOMESA_HOME}/bin"
 
     .. group-tab:: HDP
 
         Configure the environment to use an HDP install
 
         .. code-block:: bash
 
-            $ export HADOOP_HOME=/usr/hdp/current/hadoop-client/
-            $ export HBASE_HOME=/usr/hdp/current/hbase-client/
-            $ export GEOMESA_HOME=/opt/geomesa
-            $ export PATH=$PATH:$GEOMESA_HOME/bin
-
-        Optionally, add these to your bash profile or geomesa-env.sh (detailed later in
-        :ref:`setting_up_hbase_commandline`) to persist this configuration.
+            export HADOOP_HOME=/usr/hdp/current/hadoop-client/
+            export HBASE_HOME=/usr/hdp/current/hbase-client/
+            export GEOMESA_HBASE_HOME=/opt/geomesa
+            export PATH="${PATH}:${GEOMESA_HBASE_HOME}/bin"
 
     .. group-tab:: Manual Install
 
         If no HBase or Hadoop distribution is installed, try manually installing the jars from maven:
 
         .. code-block:: bash
 
-            $ export GEOMESA_HOME=/opt/geomesa
-            $ export PATH=$PATH:$GEOMESA_HOME/bin
-            $ cd $GEOMESA_HOME
-            $ bin/install-hadoop.sh lib
-            $ bin/install-hbase.sh lib
+            export GEOMESA_HBASE_HOME=/opt/geomesa
+            export PATH="${PATH}:${GEOMESA_HBASE_HOME}/bin"
+            cd GEOMESA_HBASE_HOME
+            bin/install-hadoop.sh lib
+            bin/install-hbase.sh lib
+
+        You will also need to provide the hbase-site.xml file within a directory, zip, or jar archive (the xml file
+        will not work with the Java classpath):
```

Comment:
you could mention here that the `conf` folder is on the classpath, so you can put it there